ogre2/src/Ogre2Camera.cc

```diff
@@ -387,9 +387,9 @@ void Ogre2Camera::SetVisibilityMask(uint32_t _mask)
 {
   if (_mask & ~Ogre::VisibilityFlags::RESERVED_VISIBILITY_FLAGS)
   {
-    ignerr << "Ogre2Camera::SetVisibilityMask: Mask bits " << std::hex
-           << ~Ogre::VisibilityFlags::RESERVED_VISIBILITY_FLAGS << std::dec
-           << " are set but will be ignored by ogre2 backend." << std::endl;
+    ignwarn << "Ogre2Camera::SetVisibilityMask: Mask bits " << std::hex
+            << ~Ogre::VisibilityFlags::RESERVED_VISIBILITY_FLAGS << std::dec
+            << " are set but will be ignored by ogre2 backend." << std::endl;
```

Comment:
updated comment to explain that they are ignored because of conflict with internal reserved bits used by ogre2. [5445026](https://github.com/gazebosim/gz-rendering/pull/830/commits/5445026eabf2bfb29f03ce86996833fb7d24c774)